kv/kv.go

```diff
@@ -0,0 +1,148 @@
+package kv
+
+import (
+	"bytes"
+	"crypto/sha256"
+	"encoding/json"
+	"fmt"
+	"strings"
+
+	"github.com/jordan-rash/wasmcloud-go/models"
+	"github.com/nats-io/nats.go"
+	core "github.com/wasmcloud/interfaces/core/tinygo"
+)
+
+const (
+	LINKDEF_PREFIX = "LINKDEF_"
+	CLAIMS_PREFIX  = "CLAIMS_"
+	BUCKET_PREFIX  = "LATTICEDATA_"
+)
+
+func GetKVStore(nc *nats.Conn, latticePrefix, jsDomain string) (nats.KeyValue, error) {
+	var js nats.JetStreamContext
+	var err error
+
+	if jsDomain != "" {
+		js, err = nc.JetStream(nats.PublishAsyncMaxPending(256), nats.Domain(jsDomain))
```

Comment:
Honestly, because thats what they use in the nats client examples and docs...not sure if there is a reason or not.  I can investigate is that is random or on purpose